docsite/docs/fal-serverless/isolated_functions/serving.md

```diff
@@ -0,0 +1,67 @@
+---
+sidebar_position: 5
+---
+
+# Serving functions though a managed web server
+
+Serving an `@isolated` function exposes the function through a webserver managed by `fal-serverless`.
+
+Simplest way to serve a function is to mark the function you want to serve using the `@isolated` annotation's `serve=True` option and `fal-serverless` cli command.
+
+To serve a function do the following steps:
+
+1. Mark the function you want to serve using the \@isolated annotation with the `serve=True` option:
+
+```python
+@isolated(serve=True)
+def call_str(str):
+    return str
+```
+
+2. Use the `fal-serverless` CLI command with the following syntax:
+
+```
+fal-serverless function serve ./path/to/file call_str
+>> Registered anonymous application 'd9ff88a9-6ae3-45cf-ab67-022e33e4418e'
+```
+
+You'll receive an application ID in the following format: \`XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX\`.
+
+## Access Served Function via REST API
+
+To access the served function, make a POST REST API request to the following URL:
+
+```
+https://<userid>-<application_id>.gateway.alpha.fal.ai
+```
+
+Replace `<userid>` with your user ID (e.g., `123` if your github id is `github|123`) and `<application_id>` with the application ID you received earlier. Additionally, pass your FAL key ID and key secret as headers in the request.
```

Comment:
iirc our CLI prints the URL, so maybe we could avoid this instruction to reduce complexity?

Reply:
yes yes fixed it in the new revision